test/quantization/test_quantization.py

```diff
@@ -1363,6 +1363,34 @@ def checkModelWithSequentialQuantized(self, model):
         self.assertEqual(type(model.classifier[0]), nniq.LinearReLU)
         self.assertEqual(type(model.classifier[1]), nn.Identity)
 
+    def test_fusion_conv_with_bias(self):
+        model = ModelForFusionWithBias().train()
+        # output with no fusion.
+        out_ref = model(self.img_data[0][0])
+
+        model.qconfig = QConfig(activation=torch.nn.Identity,
+                                weight=torch.nn.Identity)
+        model = fuse_modules(model, [["conv1", "bn1", "relu1"],
+                                     ["conv2", "bn2"]])
+        prep_model = prepare_qat(model, inplace=False)
+        # output with fusion but no observers.
+        out_fused = prep_model(self.img_data[0][0])
+        self.assertEqual(out_ref, out_fused)
+
+        model.qconfig = default_qat_qconfig
+        prepare_qat(model, inplace=True)
```

Comment:
It looks like this part and computation of SQNR is not used, should it be removed?